# HG changeset patch
# User Jeff Muizelaar <jmuizelaar@mozilla.com>
# Date 1276629019 14400
# Node ID b5eb246c152dedb895d3010eb7192f88cce51146
# Parent  792cd3e8aa59b24c333f2c90f938beda361b3336
cairo: PS: Add missing 'q' when resetting clip path

diff --git a/gfx/cairo/cairo/src/cairo-ps-surface.c b/gfx/cairo/cairo/src/cairo-ps-surface.c
--- a/gfx/cairo/cairo/src/cairo-ps-surface.c
+++ b/gfx/cairo/cairo/src/cairo-ps-surface.c
@@ -3696,7 +3696,7 @@ _cairo_ps_surface_set_paginated_mode (vo
     if (surface->clipper.clip.path != NULL) {
 	status = _cairo_pdf_operators_flush (&surface->pdf_operators);
 
-	_cairo_output_stream_printf (surface->stream, "Q\n");
+	_cairo_output_stream_printf (surface->stream, "Q q\n");
 	_cairo_surface_clipper_reset (&surface->clipper);
     }
 }
